### ISSUE TITLE
feature : beautiful moon phase ?

### ISSUE THREAD

**Issue by fredmo** [chubin/wttr.in#832]
Hello,

Would it be possible to have a curl command to see a beautiful moon phase picture?

Same as : 
https://theskylive.com/moon-info

![image](https://user-images.githubusercontent.com/3090107/210075996-bf4e2db1-e5e1-4d33-a8e0-589c1df5a533.png)

**Comment by dmyoung9** [comment]
From the [readme](https://github.com/chubin/wttr.in#moon-phases):

```bash
$ curl wttr.in/Moon
```

**Comment by fredmo** [comment]
yes, could it have a similar command, but with the beautiful moon as the picture?  
without ascii please?
